lib/Service/Importer/Systems/DeckJsonService.php

```diff
@@ -103,8 +104,17 @@ public function getCardAssignments(): array {
 	}
 
 	public function getComments(): array {
-		// Comments are not implemented in export
-		return [];
+		$comments = [];
+		foreach ($this->tmpCards as $sourceCard) {
+			$commentsOriginal = $sourceCard->comments;
```

Comment:
Do we need to handle cases where comments is not defined or null here? I could imagine this breaking otherwise if an old json without comments is used?

Reply:
I don't think we have to. The next loop in line 113, for my understanding,  just gets skipped if $commentsOriginal is not iterable. I tested it without comments in the json and it worked without any errors.